### PR TITLE
[completed] OMDB ~ Dubai International Airport

### PR DESCRIPTION
**Pull request by eliuuk**
Dubai International Airport is the busiest airport in terms of passenger traffic. Dubai is also home to the world-famous Emirates, low-cost carrier FlyDubai, Australian carrier Qantas, and FedEx.

### CODE REVIEW

File: assets/airports/omdb.json [hunk 1]
```diff
@@ -0,0 +1,432 @@
...
+      ["cca",       2],
+      ["afr",       2],
+      ["aic",       9],
+      ["axb",      11],
+      ["baw",       1],
+      ["cpa",       2],
+      ["ces",       2],
+      ["csn",       4],
+      ["uae",     139],
+      ["fdb",      98],
```
Comment (erikquinn) [inline assets/airports/omdb.json:309]: This airline, "FDB" has not been added to the airlines folder yet.

Comment (eliuuk) [inline assets/airports/omdb.json:309]: Hm, I'll try repushing this. Thanks!

Comment (eliuuk) [inline assets/airports/omdb.json:309]: @erikquinn Should be fixed now, I believe.

**Comment by erikquinn** [comment]
Merged via 6b84190a4ab390823dd215ad837029c5100481ad.

**Pull request closed without merging**